x/concentrated-liquidity/client/cli/tx.go

```diff
@@ -27,7 +37,7 @@ var poolIdFlagOverride = map[string]string{
 
 func NewCreateConcentratedPoolCmd() (*osmocli.TxCliDesc, *clmodel.MsgCreateConcentratedPool) {
 	return &osmocli.TxCliDesc{
-		Use:     "create-concentrated-pool [denom-0] [denom-1] [tick-spacing] [swap-fee]",
+		Use:     "create-concentrated-pool [denom-0] [denom-1] [tick-spacing] [precision-factor-At-PriceOne] [swap-fee]",
```

Comment:
Feels odd that this was missed if it was necessary – what triggered adding it?

Reply:
not sure why it was left out, the cli expected 5 args regardless